client/src/components/NavLinkDropdownMenu/NavLinkDropdownMenu.css

```diff
@@ -0,0 +1,14 @@
+.NavLinkDropdownMenu {
+  border: 2px solid gray;
+  margin: 0;
+  padding: 5px;
+  list-style: none;
+}
+.NavLinkDropdownMenu-li {
```

Comment:
You never want to put a tag name inside a css class or id - just use the tag name! So instead of something like this, just use `.NavLinkDropdownMenu li`